make/test.js

```diff
@@ -0,0 +1,33 @@
+const assert = require('assert');
+const make = require('./make.js');
+
+function sum(a, b) {
+  return a + b;
+}
+
+function multiply(a, b) {
+  return a * b;
+}
+
+describe('make', () => {
+  it('1', () => {

```

Comment:
It block should contain some description about test case.
For example:
`Should handle partial application of arguments`
`Should invoke function with memoized args when it is provided`
etc.